### ISSUE TITLE
Spelled spatial wrong... ;_;

### ISSUE THREAD

**Issue by ryanong** [ryanong/mongoid_spacial#8]
Okay so should I fix it? It is really bothering me but would break a lot of code....

**Comment by jellybob** [comment]
Yes, do it while you're pre-1.0 and can get away with a massive backwards compatibility break. I just spent a while trying to work out why I couldn't find this on Rubygems until I saw this issue.

**Comment by ryanong** [comment]
k :/ damn well there goes a lot of code. I'll try to do it next weekend when I have more time.

**Comment by jellybob** [comment]
So long as you release the gem as mongoid_spatial as well you
shouldn't cause existing code to blow up, and it'll be a fairly quick
search and replace to update.

On 23 Oct 2011, at 18:34, Ryan Ong
reply@reply.github.com
wrote:
